package.json

```diff
@@ -41,14 +41,20 @@
     "dist"
   ],
   "module": "dist/index.es.js",
+  "dependencies": {
+    "@bpmn-io/element-template-chooser": "^2.0.0",
+    "bpmn-js": "^18.6.0"
```

Comment:
We usually don't do this, as this library "builds on top of bpmn-js" anyway. If a certain bpmn-js version is needed, then we `peerDepend` on it.

Yes, we could rely on de-duplication of bundlers, but for most parts (cf. our other projects) we simply rely on bpmn-js being there, when you pull in an extension for it.